### ISSUE TITLE
ID is re-written after indexing via plugin

### ISSUE THREAD

**Issue by Pablo-Aldana** [meilisearch/strapi-plugin-meilisearch#608]
**Description**
If you create a collection on Stripi, example food

```
food: {
int id
string: name
}
```

example: 
```
 {
   id: 1, 
   name: "Pasta"
}
```
and you index it via the hooked that provides this plugin the document created on Meilisearch according with the Readme, it should be

```
"hits": [
        {
            "id": 1
            "name": "Pasta",
            "createdAt": "2022-12-07T11:38:29.336Z",
            "updatedAt": "2022-12-11T16:47:00.607Z",
            "publishedAt": "2022-12-11T16:47:00.601Z"
        }
    ],
```

However this is not the behaviour I'm experiencing,  the document created in Meilisearch is

 ```
"hits": [
        {
            "id": "food-1",
            "name": "Pasta",
            "createdAt": "2022-12-07T11:38:29.336Z",
            "updatedAt": "2022-12-11T16:47:00.607Z",
            "publishedAt": "2022-12-11T16:47:00.601Z"
        }
    ],
```

the id gets overwritten by contet-type-name+id 

This leads to fail on final application where is needed the Strapi field id with the original int value.

If I create the document via curl this does not happens and the document is created correctly.

**Expected behavior**

Document id shouldn't be changed by Meilisearch and should remain as its original.

**Current behavior**
Id is written on a custom format "content-type-name+id"

Steps to reproduce:

- Install Strapi
- Install Meilisearch plugin 
- Create a component and add a record
- Go to the Meilisearch plugin and select the component to be indexed
- The document is created with wrong ID.

**Environment (please complete the following information):**
- OS: [e.g. Debian GNU/Linux] MAC OS Ventura 13 using Docker
- Meilisearch version: v.0.30.2 and v.0.30.0
- strapi-plugin-meilisearch version: 0.8.2
- Strapi version: v4.5.3

Additional Info: After reviewing the code seems to be intentionally made on the file adatar.js
  ```
addCollectionNamePrefix: function ({ contentType, entries }) {
      return entries.map(entry => ({
        ...entry,
        id: this.addCollectionNamePrefixToId({
          entryId: entry.id,
          contentType,
        }),
      }))
    },
```

Which in my opinion from my ignorance on the code, is an error as whoever is searching might need the ID to use the document. i.e: I search for foods to add it to an order to later in the backend update the stock, which won't be possible as the ID was change by meilisearch.
I do believe the document should remain intact, and if there is a need on meilisearch to have different ID it should be called differently i.e _id or _mid ... Actually curl options does not have this limitation

However in the documentation of the plugin it does represent the id without transformation which leads to misinformation 
<img width="1072" alt="image" src="https://user-images.githubusercontent.com/9969839/206919102-6f35c2e3-4de3-4614-b619-f1757add361b.png">

**Comment by bidoubiwa** [comment]
Hey @Pablo-Aldana 
Unfortunately this is an expected behavior. I need to overwrite the ID in case you want to merge two collections into one index.

For example

```
food: {
int id
string: name
}


desert: {
   int: id
   string: name
}
```

If you decide to put them both in the same index:

```
// config/plugins.js

module.exports = () => ({
  //...
  meilisearch: {
    config: {
      food: {
        indexName: 'menu',
      },
      desert: {
        indexName: 'menu',
      },
    },
  },
})
```

I need to ensure that the `id: 1` of one of your food is not overwriting the `id: 1` of one of your desert.
To avoid this, I prepended every `id` with its collection name.

If you want a quick solution to this issue, I would suggest your suggested solution where you create a custom field `_id` (or something else) with the original value of your `id` in `transformEntry`. 

Nonetheless

> I do believe the document should remain intact, and if there is a need on meilisearch to have different ID it should be called differently i.e _id or _mid ... Actually curl options does not have this limitation

I think it is a great idea :) I'll create an issue with this suggestion. 
Updating the documentation on the unexpected behavior would also be great!

**Comment by bidoubiwa** [comment]
See #609

**Comment by bidoubiwa** [comment]
Fixed by #612
